static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -355,6 +403,77 @@ export function getMetricAlertChartOption({
       });
   }
 
+  if (anomalies) {
+    const anomalyBlocks: any[] = [];
+    let start: string | undefined;
+    let end: string | undefined;
+    anomalies
+      .filter(anomalyts => {
+        const ts = new Date(anomalyts.timestamp).getTime();
+        return firstPoint < ts && ts < lastPoint;
+      })
+      .forEach(anomalyts => {
+        const {anomaly, timestamp} = anomalyts;
+
+        if (
+          [AnomalyType.high, AnomalyType.low].includes(anomaly.anomaly_type as string)
```

Comment:
if its not merged yet anomalyType would be preferred but we miss this quite a bit in various endpoints
```suggestion
          [AnomalyType.high, AnomalyType.low].includes(anomaly.anomalyType as string)
```

Reply:
Oh.
actually, would be dependent on the API response i guess.
I won't merge this until i've verified the api response